content/javascript/concepts/strings/terms/padEnd/padEnd.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.padEnd()'
+Description: 'Pads a string with a given string several times until it reaches the given length.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, the **padEnd()** method is used with javascript strings to pad the end of a string with another string, until it reaches a specified length. It was introduced with ECMAScript 2017.
+
+## Syntax
+
+```pseudo
+string.padEnd(length, padStr)
+```
```

Comment:
```suggestion
```pseudo
string.padEnd(length, padStr)
```
```

Reply:
I don't understand what the difference is here? @SaviDahegaonkar